### DISCUSSION TITLE
error when compiling to exe

### DISCUSSION THREAD

**Discussion by rsnowakowski** [exaloop/codon#274] [Q&A] [answered]
using a standard ubuntu (22.10 install)

codon sample program fib.py

from time import time
def fib(n):
    return n if n < 2 else fib(n - 1) + fib(n - 2)
t0 = time()
ans = fib(40)
t1 = time()
print(f'Computed fib(40) = {ans} in {t1 - t0} seconds.')

codon run fib.py works fine

but
codon build -exe fib.py returns error:

/usr/bin/ld: cannot find -lz: No such file or directory
collect2: error: ld returned 1 exit status
error: process for 'gcc' exited with status 1

**Comment by Chasmccoy** [comment]
Same issue with Ubuntu.  Codon does create the object. The linker appears to be the issue.

**Comment by elisbyberi** [comment]
This may be a duplicate of #281

**Reply by Chasmccoy** [reply, marked as answer]
You are so absolutely correct.  Fixed the problem.  Thank you.
